tests/test_vcs_git/test_vcs_git.py

```diff
@@ -0,0 +1,66 @@
+from os.path import dirname, join
+import subprocess
+
+from dxr.testing import DxrInstanceTestCase
+
+from nose import SkipTest
+from nose.tools import ok_
+
+
+class GitTests(DxrInstanceTestCase):
+    """Test our Git integration, both core and omniglot."""
+
+    @classmethod
+    def setup_class(cls):
+        """git does not allow us to commit the .git folder under the build
+        directory, so we extract it before the tests.
+
+        """
+        build_dir = join(dirname(__file__), 'code')
+        subprocess.check_call(['make'], cwd=build_dir)
```

Comment:
I wanted to do this, but unfortunately build.py scans for Vcs repositories before building. However it is in the build that we create the Vcs repository itself.